src/inc/til.h

```diff
@@ -11,8 +11,8 @@
 #include "til/some.h"
 #include "til/size.h"
 #include "til/point.h"
-#include "til/rectangle.h"
 #include "til/operators.h"
+#include "til/rectangle.h"

```

Comment:
y?

Reply:
Because I took the imports out of the top of all of the til headers and am letting this be the correct order of import, not any individual one. I needed the operators in the rectangle but the operators and rectangle were circularly including each other. So I took the circle out and this is it.